src/NzbDrone.Common.Test/Http/HttpHeaderFixture.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using NzbDrone.Test.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NzbDrone.Common.Http;
+using System.Collections.Specialized;
+
+namespace NzbDrone.Common.Test.Http
+{
+    [TestFixture]
+    public class HttpHeaderFixture : TestBase
+    {
+        [TestCase("text/html; charset=\"utf-8\"")]
+        [TestCase("text/html; charset=utf-8")]
+
+        public void should_get_encoding(string contentType)
+        {
+            var headers = new NameValueCollection();
+
+            headers.Add("content-type", contentType);
+
+            var httpheader = new HttpHeader(headers);
+
+            Assert.DoesNotThrow(() => httpheader.GetEncodingFromContentType());
```

Comment:
That would be a separate test `should_throw_when_invalid_encoding_is_in_header` (or something similar) to that.